Sources/IDEUtils/generated/SyntaxClassification.swift

```diff
@@ -69,306 +70,308 @@ extension SyntaxClassification {
-        return (.buildConfigId, false)
-      case (.ifConfigDecl, 3):
-        return (.buildConfigId, false)
-      case (.declModifier, 1):
+      case (.attribute, 3): 
         return (.attribute, false)
-      case (.operatorDecl, 7):
-        return (.operatorIdentifier, false)
-      case (.precedenceGroupRelation, 1):
+      case (.availabilityVersionRestriction, 1): 
         return (.keyword, false)
-      case (.precedenceGroupAssociativity, 1):
+      case (.constrainedSugarType, 1): 
         return (.keyword, false)
-      case (.attribute, 3):
+      case (.declModifier, 1): 
         return (.attribute, false)
-      case (.forInStmt, 5):
+      case (.expressionSegment, 5): 
+        return (.stringInterpolationAnchor, true)
+      case (.expressionSegment, 9): 
+        return (.stringInterpolationAnchor, true)
+      case (.forInStmt, 5): 
         return (.keyword, false)
-      case (.simpleTypeIdentifier, 1):
-        return (.typeIdentifier, false)
-      case (.memberTypeIdentifier, 5):
+      case (.ifConfigClause, 1): 
+        return (.buildConfigId, false)
+      case (.ifConfigDecl, 3): 
+        return (.buildConfigId, false)
+      case (.memberTypeIdentifier, 5): 
         return (.typeIdentifier, false)
-      case (.constrainedSugarType, 1):
+      case (.operatorDecl, 7): 
+        return (.operatorIdentifier, false)
+      case (.precedenceGroupAssociativity, 1): 
         return (.keyword, false)
-      case (.availabilityVersionRestriction, 1):
+      case (.precedenceGroupRelation, 1): 
         return (.keyword, false)
-      default: return nil
+      case (.simpleTypeIdentifier, 1): 
+        return (.typeIdentifier, false)
+      default: 
+        return nil
       }
     } else {
       switch (parentKind, indexInParent) {
-      case (.ifConfigClause, 3):
-        return (.buildConfigId, false)
-      case (.operatorDecl, 3):
+      case (.backDeployVersionArgument, 1): 
+        return (.keyword, false)
+      case (.customAttribute, 3): 
         return (.attribute, false)
-      case (.customAttribute, 3):
+      case (.ifConfigClause, 3): 
+        return (.buildConfigId, false)
+      case (.operatorDecl, 3): 
         return (.attribute, false)
-      case (.backDeployVersionArgument, 1):
-        return (.keyword, false)
-      default: return nil
+      default: 
+        return nil
       }
     }
   }
 }
 
 extension RawTokenKind {
   internal var classification: SyntaxClassification {
-    switch (self) {
-    case .associatedtypeKeyword:
+    switch self {
+    case .associatedtypeKeyword: 
       return .keyword
-    case .classKeyword:
+    case .classKeyword: 
       return .keyword
-    case .deinitKeyword:
+    case .deinitKeyword: 
       return .keyword
-    case .enumKeyword:
+    case .enumKeyword: 
       return .keyword
-    case .extensionKeyword:
+    case .extensionKeyword: 
       return .keyword
-    case .funcKeyword:
+    case .funcKeyword: 
```

Comment:
Nyt sure why the parser adds a whitespace after the `:` as there is no whitespace in the generator 
Might be bug @ahoppen ?

Reply:
I think formatting does because `:` is considered a punctuator and we want the whitespace in e.g. `[String: Int]`. I think it’s fine if we just keep the whitespace. It doesn’t really do any harm.